scripts/jenkins/qa/cloud-mkphyscloud-qa-test-horizon-ui-integration.yaml

```diff
@@ -0,0 +1,71 @@
+- job:
+    name: cloud-mkphyscloud-qa-test-horizon-ui-integration
+    node: cloud-mkphyscloud-gate-qa
+    description: |
+      Runs tempest smoke tests
+      Useful only for cloud that was built by mkcloud.
+      Mandatory parameter: hw_number
+
+    logrotate:
+      numToKeep: 7
+      daysToKeep: -1
+
+    wrappers:
+      - build-name:
+          name: '#${BUILD_NUMBER} - ${scenario_name} - qa$hw_number - horizon-ui'
```

Comment:
`horizon` instead `horizon-ui`, the shorter the build name the better it's readable in the jenkins UI.